### PR TITLE
Aida

### PR DESCRIPTION
**Pull request by nichubbard**
mostly complete Aida analysis routines
front-back matching and AidaHit

incomplete online spectra still

Scalers are unpakced (for time machien)

### CODE REVIEW

File: CMakeLists.txt
Comment (cej25) [inline CMakeLists.txt:1]: This AIDA folder contains code that we're placing in c4source for the other subsystems right? i.e. c4root/c4source/aida. Is it possible to move it there instead?

Comment (nichubbard) [reply CMakeLists.txt:1]: Not sure - I based it off R3BRoot where things like calib/analysis is in a detector folder, not really part of the ucesb/source bit

But can move to c4source easily

**Comment by nichubbard** [comment]
Performance seems absurdly slow - investigating (if I can build fairsoft and fairroot)
the ucesb unpacking is fast but something in c4root analysis slows it down

**Comment by nichubbard** [comment]
Contents of 'aida' moved to c4source or c4data folders 

Performance improved by factor 3 or so;  more optimisations would be trickier

**Review by cej25 — APPROVED** [review]

**Pull request merged**